app/models/grda_warehouse/system_cohorts/currently_homeless.rb

```diff
@@ -33,43 +33,60 @@ def sync(processing_date: Date.current, date_window: 1.day)
       # if exit was to permanent destination within 2 year range, or move in was within 2 year range  - Returned from housing
       # else Returned from inactive unless no service in INACTIVE period
 
-      moved_in_ph = enrollment_source.
-        ongoing.
-        ph.
+      moved_in_ph = enrollment_source.ph.
+        ongoing(on_date: @processing_date).
         where(she_t[:move_in_date].lt(@processing_date)).
         select(:client_id)
```

Comment:
Only `ph` enrollments should have move in dates, but I think we should put that scope back in in case of bad data?

Reply:
`ph` is there on line 36, move-in is required on line 38.  Am I missing something?